introduction.md

```diff
@@ -1,3 +1,9 @@
 Hi Im Aranza and I like to sleep :D
 
-My favorite Code School path is Ruby.
\ No newline at end of file
+My favorite Code School path is Ruby.
+
+Deadly skills
+*Git
+*Golang
+*Ruby
+*Java
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?